### DISCUSSION TITLE
Fetch changelog of container images used in a Helm chart

### DISCUSSION THREAD

**Discussion by akselleirv** [renovatebot/renovate#24391] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

v39.0.3

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

When updating a Helm chart I not only care about the release notes for the helm chart release itself, but mostly about the release notes for the container images the helm chart is packaging. 

For example, I'm getting a new PR for a update of the [loki-distributed](https://grafana.github.io/helm-charts) chart changelog. But I also want that the PR includes the release notes for the appVersion. 

Is this possible?

### What you tried so far.

I'm only getting the helm chart repository release notes.

**Comment by HonkingGoose** [comment]
Can you link to the PR you're getting from Renovate? So we can see what's missing?

With npm packages, sometimes the _maintainer of the package_ must put a `repository` field in their `package.json` file, so that Renovate knows where to get the changelogs. Something similar may be needed for your Helm chart dependency...

**Reply by MisterWS** [reply]
Here's an example of a PR from a while ago (on someone else' repo - but the format is identical): https://github.com/GiorgioAresu/homelab-gitops/pull/1146

The chart release notes are empty: https://github.com/longhorn/charts/releases/tag/longhorn-1.4.0
But the main repo has full release notes: https://github.com/longhorn/longhorn/releases/tag/v1.4.0

Is there any way to link / provide Renovate with a 'translation' between the Helm chart and the underlying container/GitHub release?

**Reply by jamietanna** [reply]
You might be able to use the `sourceUrl` config to overwrite it (inside `packageRules`) for that package? (I don't believe it's explicitly documented as a way to do this, but https://docs.renovatebot.com/key-concepts/changelogs/ reads like that may work)
